Description: Updated makefile
nothing of real interest
 .
 dirt (1.0.0-1) unstable; urgency=low
 .
   * Initial release
Author: Alex McLean <alex@slab.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- dirt-1.0.0.orig/Makefile
+++ dirt-1.0.0/Makefile
@@ -12,4 +12,10 @@ dirt: dirt.o jack.o audio.o file.o serve
 	$(CC) dirt.o jack.o audio.o file.o server.o segment.o $(CFLAGS) $(LDFLAGS) -o dirt 
 
 test : test.c Makefile
-	$(CC) test.c -llo -o test
\ No newline at end of file
+	$(CC) test.c -llo -o test
+
+install: dirt
+	install -d $(DESTDIR)/bin/
+	install -m 0755 dirt $(DESTDIR)/bin/dirt
+
+
